packages/office-addin-debugging/src/start.ts

```diff
@@ -289,6 +290,15 @@ export async function startDebugging(manifestPath: string, appType: AppType, app
             }
         }
 
+        if (isWebAppType && documentUrl !== undefined && devServerPort !== undefined) {
+            try {
+                console.log(`Sideloading the Office Add-in to web...`);
+                await sideloadWebAddin(documentUrl, manifestPath, devServerPort)
```

Comment:
I would really like to see sideloading a bit more unified between web and desktop. I haven't looked closely at the code here in a while though. Sideload is part of the manifest or dev-settings package and I think this should be true whether for desktop or web sideloading.

We should also discuss being able to specify the document when doing desktop sideload as well.